uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -1,27 +1,39 @@
 package com.learning.uwuno;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
Separate from the uwuno packages?

Reply:
Ahhhhh other way around should keep all the uwuno ones on top like all the other files.